src/components/settings-security-pin/settings-security-pin.tsx

```diff
@@ -0,0 +1,57 @@
+import React, {useCallback, useRef, useState} from 'react';
+
+import {Pin, PinInterface} from '@app/components/pin';
+import {hideModal, sendNotification, showModal} from '@app/helpers';
+import {useApp, useWallets} from '@app/hooks';
+import {I18N, getText} from '@app/i18n';
+
+interface SettingsSecurityPinProps {
+  goBack?: () => void;
+}
+
+export const SettingsSecurityPin = ({
+  goBack = () => {},
+}: SettingsSecurityPinProps) => {
+  const app = useApp();
+  const wallets = useWallets();
+  const pinRef = useRef<PinInterface>();
+  const [pin, setPin] = useState('');
+  const [isRepeatPin, setIsRepeatPin] = useState(false);
+  const onPin = useCallback((newPin: string) => {
+    setPin(newPin);
+    setIsRepeatPin(true);
+  }, []);
+
+  const onPinRepeated = useCallback(
+    async (repeatedPin: string) => {
+      if (pin === repeatedPin) {
+        showModal(getText(I18N.settingsSecurityPinLoading));
```

Comment:
Translations are not needed here, because this is the name of the Modal that identifies